include/exadg/poisson/spatial_discretization/operator.h

```diff
@@ -133,6 +133,9 @@ class Operator : public dealii::Subscriptor
                      VectorTypeDouble const &                         src) const;
 #endif
 
+  unsigned int
```

Comment:
Could you add a TODO here "// TODO: we currently need this function public for precice-based FSI" for example. The same below.